amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2495,7 +2497,193 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  ##retention-schedule
+  '/categories/{categoryId}/retention-schedules':
+    post:
+      tags:
+        - retention-schedule
+      summary: Create a retention schedule
+      description: |
+        Create a retention schedule.        
+        For example, the following JSON body to create a retention schedule:
```

Comment:
```suggestion
        For example, using the following JSON body will create a retention schedule:
```

Reply:
done the changes as per the review comment.